From d9eb91da5e4a20aec11f94c9579e8be07e130220 Mon Sep 17 00:00:00 2001
From: danie007 <danilselvan@gmail.com>
Date: Mon, 6 Dec 2021 13:29:31 +0530
Subject: [PATCH] Enable QSPI boot support

Added configuration options to boot/load U-Boot from the QSPI flash

Signed-off-by: danie007 <danilselvan@gmail.com>
---
 configs/mx6sxsabresd_optee_defconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/configs/mx6sxsabresd_optee_defconfig b/configs/mx6sxsabresd_optee_defconfig
index 7945bb639c..ee4e36ac05 100644
--- a/configs/mx6sxsabresd_optee_defconfig
+++ b/configs/mx6sxsabresd_optee_defconfig
@@ -1,3 +1,8 @@
+# Added by Yocto to support QSPI Boot
+CONFIG_QSPI_BOOT=y
+CONFIG_ENV_IS_IN_SPI_FLASH=y
+CONFIG_ENV_SECT_SIZE=0x10000
+
 CONFIG_ARM=y
 CONFIG_ARCH_MX6=y
 CONFIG_NR_DRAM_BANKS=1
